tools/operator/launcher/k8s/network.go

```diff
@@ -2,241 +2,522 @@ package k8s
 
 import (
 	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+
+	v12 "k8s.io/api/core/v1"
+	apiResource "k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
 
 	"github.com/hyperledger/fabric-test/tools/operator/launcher/nl"
 	"github.com/hyperledger/fabric-test/tools/operator/logger"
-	"github.com/hyperledger/fabric-test/tools/operator/networkclient"
+
+	"github.com/hyperledger/fabric-test/tools/operator/fabricconfig"
 	"github.com/hyperledger/fabric-test/tools/operator/networkspec"
 	"github.com/hyperledger/fabric-test/tools/operator/paths"
-	"github.com/pkg/errors"
 )
 
+//K8s -
 type K8s struct {
 	KubeConfigPath string
 	Action         string
 	Arguments      []string
 	Config         networkspec.Config
+	Launch         []LaunchConfig
+}
+
+//LaunchConfig --
+type LaunchConfig struct {
+	Name       string
+	Type       string
+	Containers []v12.Container `json:"containers,omitempty"`
+	Volumes    []v12.Volume    `json:"volumes,omitempty"`
+	Ports      []int32         `json:"ports,omitempty"`
 }
 
-func (k K8s) Args() []string {
+func (k8s K8s) k8sLaunchObject(nsConfig networkspec.Config) ([]LaunchConfig, error) {
+
+	launchConfig := []LaunchConfig{}
```

Comment:
Pointer declarations are of the form `var launchConfig []LaunchConfig`

Reply:
agreed and fixed.